_posts/2019/2019-09-20-linux-party-experience.md

```diff
@@ -0,0 +1,158 @@
+
+Now for all this blabbering from our experience a general space ratio of 40:60 between / and /home and having swap as double the size of your RAM is the ideal composition. Also, root requires a minimum of 20 GB for smooth operation
+
+## The actual partitioning
+
+Now we have three cases
+
+* Only SSD or only HDD
+* SSD + HDD
+* Optane + HDD
+
+Now for only HDD laptops, the work is pretty straightforward that you have to shrink the required memory which you have calculated previously. 
+
+For only SSD the process is the same as only HDD laptops except that you have to shift your SSD from RAID to AHCI using [this](https://support.thinkcritical.com/kb/articles/switch-windows-10-from-raid-ide-to-ahci). Step 5 of the previous link will vary with manufacturer, so explore your BIOS a bit (don't go on changing any random setting!) and you'll be bound to hit it or you can google for "____ RAID to AHCI".
+
+For SSD + HDD laptops you would want to keep minimum stuff on SSD as it is generally small. So what should be done in this case is to allocate space for / on the SSD(minimal boot time) and /home on the HDD.
+
+For shrinking your Windows drives follow [this guide](https://www.dummies.com/computers/pcs/how-to-shrink-a-hard-drive-volume-in-windows/) for both SSD and HDD
+
+Optane is rare so first check [this](https://www.intel.in/content/www/in/en/support/articles/000023990/memory-and-storage/intel-optane-memory.html) to determine if you have Optane in use in case you are unsure. Installing Ubuntu on Optane is a bit complex so we won't be going there, use [this](https://support.cyberpowerpc.com/hc/en-us/articles/360014775073-How-do-I-disable-Intel-Optane-)(also Disable Intel Optane memory with Intel RST is enough, no need disable from BIOS!), install ubuntu but remember to keep EXTRA 10 MB unallocated space at the END of your HDD which is required by Optane, other than the memory used by Ubuntu. Then use the previous guide to reenable Optane.
+
+Use [this](https://askubuntu.com/a/521195) to apply the theory you learned. "/dev\*" type of names refer to HDD and "/dev/nvmp\*" refer to SSD.
+
+In `Device for boot loader installation` shown in the previous link select your SSD in case you have one or you HDD in other case.
+
+# GRUB
+
+Remember the talk about bootloaders? now GRUB is your new bootloader. You would be seeing a black screen with white text in a not-so-appealing font show up after your manufacturer's logo on startup. Here is where you can select your OS using arrow keys.
+
+In case GRUB is now showing up and you are directly booting into an OS(which will not happen in case you followed the instructions) then the issue can be subjective and your best bet would be to do subjective google searches describing what is happening and we are 99% sure that you will find a solution(trust us we've been there). You can also reach out to us [here](chat.sdslabs.co) and we would love to help you!
```

Comment:
```suggestion
In case GRUB is now showing up and you are directly booting into an OS(which should not happen if you followed the instructions), the issue can be subjective and your best bet would be to do google searches describing what is happening and you should find a solution(trust us, we've been there too). You can also reach out to us [here](chat.sdslabs.co) and we would love to help!
```